sdk/search/ci.yml

```diff
@@ -6,6 +6,10 @@ resources:
     - repository: azure-sdk-build-tools
       type: github
       name: internal/azure-sdk-build-tools
+    - repository: azure-sdk-tools
```

Comment:
@chidozieononiwu looks like another yml file missed. please double check the rest.

Reply:
After a quick [search](https://github.com/Azure/azure-sdk-for-java/search?p=1&q=repository%3A+azure-sdk-build-tools&unscoped_q=repository%3A+azure-sdk-build-tools) I can confirm that the azure-sdk-tools repository is present on all 20 ci.yml files. Apologies for the earlier missing ones.